python/cudf/cudf/_lib/copying.pyx

```diff
@@ -745,7 +742,7 @@ def segmented_gather(Column source_column, Column gather_map):
 cdef class _CPackedColumns:
 
     @staticmethod
-    cdef _CPackedColumns from_py_table(Table input_table, keep_index=True):
+    def from_py_table(input_table, keep_index=True):
```

Comment:
Does changing this from `cdef` to `def` break anything? An alternative to this is 
```suggestion
    cdef _CPackedColumns from_py_table(object input_table, keep_index=True):
```

Reply:
No, changing it doesn't break anything. I actually had to change it because Cython was having trouble with the calling convention when I tried passing it as an object. I don't know exactly why and didn't investigate further since we don't really need this to be a cdef function. All the places that call this function are pure Python code, so returning a typed object here doesn't do anything because it immediately decays to a Python object pointer anyway.